easylist_adult/adult_adservers.txt

```diff
@@ -108,6 +108,7 @@
 ||festaporno.com^$third-party
 ||filexan.com^$third-party
 ||findandtry.com^$third-party
+||fireadsone.com^$third-party
```

Comment:
No need for third-party

![image](https://user-images.githubusercontent.com/44526987/134208677-e4d8d689-2c31-4228-ab30-98b37c70ee0c.png)